src/panel/pages/events/components/TimelinePane/TimelinePane.tsx

```diff
@@ -61,61 +61,65 @@ const EventSection: FC<{ event: DebugEvent }> = ({ event }) => {
   return (
     <PaneSection>
       <Pane.Header>Event</Pane.Header>
-      <Pane.Item>
-        <Pane.ItemTitle>Event</Pane.ItemTitle>
-        <p>{event.type}</p>
-      </Pane.Item>
-      <Pane.Item>
-        <Pane.ItemTitle>Message</Pane.ItemTitle>
-        <p>
-          <Icon icon={faQuoteLeft} />
-          {event.message}
-        </p>
-      </Pane.Item>
-      <Pane.Item>
-        <Pane.ItemTitle>Timestamp</Pane.ItemTitle>
-        <p>
-          <Icon icon={faStopwatch} />
-          {timestamp}
-        </p>
-      </Pane.Item>
-      {event.data && (
+      <ScrollArea>
```

Comment:
Non blocking - how about

```suggestion
      <Pane.Body>
```